Header header
string essid
float32 frequency
float32 signal_level

time transmit_time